apps/frontend-manage/src/components/common/ColorPicker.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Label, ThemeContext } from '@uzh-bf/design-system'
+import { useContext, useState } from 'react'
+import { HexColorInput, HexColorPicker } from 'react-colorful'
+
+interface Props {
+  color: string
+  onSubmit: (newColor: string) => void
+  onAbort: () => void
+}
+
+function ColorPicker({ color, onAbort, onSubmit }: Props) {
+  const theme = useContext(ThemeContext)
+  const [newColor, setNewColor] = useState(color)
+
+  const presetColors = ['#262FAD', '#016272', '#5FB1F9', '#FE7408', '#D84B39']
```

Comment:
Extract to constants.ts and import here to make it more accessible.

Reply:
@BShaq There is a constants file in the `shared-components` package, which we can  use to have access to those variables in all different frontends :) You can find it at `packages/shared-components/src/constants.ts`

Reply:
@sjschlapbach Thanks for the hint! Adjusted the code accordingly :)

Reply:
Perfect 👍